src/components/ProgressBar/ProgressBar.js

```diff
@@ -26,7 +26,15 @@ class ProgressBar extends Component<Props> {
 
     return (
       <Wrapper height={height}>
-        <Motion style={{ progress: spring(progress, { stiffness, damping }) }}>
+        <Motion
+          style={{
+            progress: spring(progress, {
+              stiffness,
+              damping,
+              precision: 0.0001,
```

Comment:
This is an unrelated change, made clear by the story; because progress oscillates from `0` to `1`, the default precision of `0.001` meant that it would actually truncate the animation too soon. So I set a smaller value, which works better for the range of values specified.